src/Core/Components/DateTime/FluentDatePicker.razor.cs

```diff
@@ -39,7 +41,13 @@ public FluentDatePicker()
 
     protected override string? FormatValueAsString(DateTime? value)
     {
-        return Value?.ToString(Culture.DateTimeFormat.ShortDatePattern, Culture);
+        return Value?.ToString(View switch
```

Comment:
This code is duplicated with `FormatAccardingToView` method

Reply:
Added two unit tests to verify that placeholder get correct value